checks/cs50/2017/x/resize/less/checks.py

```diff
@@ -0,0 +1,65 @@
+import os
+import sys
+
+sys.path.append(os.getcwd())
+from check50 import *
+
+class Resize_Less(TestCase):
+
+    @check()
+    def resize_bmp_exist(self):
+        """resize.c and bmp.h exist."""
+        super().exists("resize.c")
+        super().exists("bmp.h")
+
+    @check("resize_bmp_exist")
+    def compiles(self):
+        """resize.c compiles."""
+        self.spawn("clang -o resize resize.c -lcs50").exit(0)
+
+    @check("compiles")
+    def scale_by_1(self):
+        """doesn't resize 1x1-pixel BMP when n is 1"""
+        self.include("1.bmp")
+        child = self.spawn("./resize 1 1.bmp outfile.bmp")
+        self.spawn("diff 1.bmp outfile.bmp").stdout("", "").exit(0)
+
+    @check("compiles")
+    def scale_by_2(self):
+        """resizes 1x1-pixel BMP to 2x2 correctly when n is 2"""
+        self.include("1.bmp")
+        self.include("2.bmp")
+        child = self.spawn("./resize 2 1.bmp outfile.bmp")
+        self.spawn("diff 2.bmp outfile.bmp").stdout("", "").exit(0)
```

Comment:
I believe it means we are expecting no output i.e. diff outputs nothing. The exit code is probably a better indicator though. `diff` returns a 0 when the files are identical, 1 when they're different or a 2 on error (file not found).